homeassistant/components/airnow/config_flow.py

```diff
@@ -98,6 +98,41 @@ async def async_step_user(self, user_input=None):
             errors=errors,
         )
 
+    @staticmethod
+    @core.callback
+    def async_get_options_flow(
+        config_entry: config_entries.ConfigEntry,
+    ) -> config_entries.OptionsFlow:
+        """Create the options flow."""
+        return OptionsFlowHandler(config_entry)
+
+
+class OptionsFlowHandler(config_entries.OptionsFlow):
+    """Handle an options flow for AirNow."""
+
+    def __init__(self, config_entry: config_entries.ConfigEntry) -> None:
+        """Initialize options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(self, user_input):
+        """Manage the options."""
+        if user_input is not None:
+            return self.async_create_entry(title="", data=user_input)
+
+        return self.async_show_form(
+            step_id="init",
+            data_schema=vol.Schema(
+                {
+                    vol.Required(
```

Comment:
```suggestion
                    vol.Optional(
```

Reply:
Related to the above about the API always requiring a value.  I think after the initial configuration flow, treating the distance as a required parameter (possibly set to a default value during initial configuration) is the best way to avoid API errors.

Reply:
As you have a default if the user finds a way to push `None` into the field it will use the default number.
Essentially if you have a default it should be `Optional` as the user is then allowed to skip the field and it will use default value.

Reply:
OK, makes sense.  Changed in the latest commit.